frontend/src/libs/import-export/prepare-data.ts

```diff
@@ -90,9 +104,12 @@ export function prepareDdahDataFactory(ddahFilter: FilterFunc<Ddah>) {
         }
         return dataToFile(
             {
-                toSpreadsheet: () => prepareSpreadsheet.ddah(ddahs),
+                toSpreadsheet: () =>
+                    spreadsheetUndefinedToNull(prepareSpreadsheet.ddah(ddahs)),
                 toJson: () => ({
-                    ddahs: ddahs.map((ddah) => prepareMinimal.ddah(ddah)),
+                    ddahs: jsonUndefinedToNull(
```

Comment:
`jsonUndefinedToNull` should be called inside the `prepareMinimal`/`prepareSpreadsheet` functions.

Reply:
`spreadsheetUndefinedToNull` is moved inside `prepareSpreadsheet` functions.